src/marqo/tensor_search/api.py

```diff
@@ -185,6 +185,10 @@ def add_or_replace_documents(
         ),
         mappings: typing.Optional[dict] = Depends(api_utils.decode_mappings)):
     """add_documents endpoint (replace existing docs with the same id)"""
+    if processes is not None:
+        raise InvalidArgError("The parameter `process` is no longer supported anymore. Please check the documentation for more information. \n"
```

Comment:
Parameter is `processes` (plural). Also I don't think backticks will be picked up in error message i.e. won't affect formatting. 

Could probably do without the line break. Error messages are generally one paragraph

Reply:
updated

Reply:
@farshidz We use backticks as quotation marks in our error messages, following the style found in other projects, like Meilisearch: https://www.meilisearch.com/docs/reference/errors/overview#error-format